reference/configuration.html.md.erb

```diff
@@ -291,15 +291,18 @@ Times are in milliseconds unless units are specified.
 
 ## The `services` sections
 
-The `services` sections configure the mapping of ports on the application to ports and services on Fly.io. These mappings determine how connections to the application will be handled on their journey from the Fly.io edge network to running Fly Apps.
+The `services` sections define how Fly Proxy connects requests to services running within Machines, via an app's public [Anycast](/docs/reference/services/) or private [Flycast](/docs/reference/private-networking/#flycast-private-load-balancing) address. They also configure additional settings tied to Fly Proxy behavior for a service. If a service only needs HTTP and HTTPS on standard ports, you can use the less verbose [`http_service`](#the-http_service-section).
```

Comment:
I like that we bring Fly Proxy in here!
What does "via" mean in this case?
I'm not sure if the second sentence is needed. The "they" and the meaning of "Fly Proxy behaviour" isn't super clear.

Reply:
```suggestion
The `services` sections define how Fly Proxy connects requests that hit an app's public [Anycast](/docs/reference/services/) or private [Flycast](/docs/reference/private-networking/#flycast-private-load-balancing) address to services running within Machines. These sections also configure additional settings tied to Fly Proxy behavior for a service. If a service only needs HTTP and HTTPS on standard ports, you can use the less verbose [`http_service`](#the-http_service-section).
```

Better?